pkg/apis/core/types_cloudprofile.go

```diff
@@ -150,3 +152,19 @@ const (
 	// VolumeClassPremium is a constant for the premium volume class.
 	VolumeClassPremium string = "premium"
 )
+
+// VersionClassification is the logical state of a version according to https://github.com/gardener/gardener/blob/master/docs/proposals/05-versioning-policy.md
```

Comment:
Can you please use a link to the documentation you wrote instead of the Proposal?
```suggestion
// VersionClassification is the logical state of a version according to https://github.com/gardener/gardener/blob/master/docs/operations/versioning.md
```